src/Tribe/Views/V2/View.php

```diff
@@ -1039,9 +1046,25 @@ protected function setup_template_vars() {
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
 			],
 			'messages'          => $this->get_messages( $events ),
+			'breadcrumb_header' => [],
 			'start_of_week'     => get_option( 'start_of_week', 0 ),
 		];
 
+		/**
+		 * @todo  @bordoni move this to a template vars modifier of sorts, pending convo with @lucatume
+		 */
+		if ( ! empty( $this->context->get( 'taxonomy' ) ) && ! empty( $this->context->get( $this->context->get( 'taxonomy' ) ) ) ) {
+			$term = get_term_by( 'slug', $this->context->get( $this->context->get( 'taxonomy' ) ), $this->context->get( 'taxonomy' ) );
+
+			if ( ! empty( $term ) ) {
+				$template_vars['breadcrumb_header'][] = [
+					'priority' => 10,
+					'slug' => 'taxonomy-' . $this->context->get( 'taxonomy' ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'slug'" and double arrow, but found 1.

<sub>hash: 2ff00361d4ec5cba8dcaac4336eb7e66</sub>